src/google/protobuf/stubs/port.h

```diff
@@ -241,9 +241,9 @@ static const uint64 kuint64max = GOOGLE_ULONGLONG(0xFFFFFFFFFFFFFFFF);
 #endif
 #endif
 
-#ifndef GOOGLE_PROTOBUF_ATTRIBUTE_RETURNS_NONNULL
+#ifndef GOOGLE_PROTOBUF_ATTRIBUTE_RETURNS_NONnullptr
```

Comment:
Here is a spot where I think the search-and-replace made an unnecessary change.

Reply:
Ack